Soundex.test.py

```diff
@@ -1,14 +1,31 @@
 import unittest
-from Soundex import generate_soundex
-
+from Soundex import get_soundex_code, generate_soundex
+ 
 class TestSoundex(unittest.TestCase):
-
-    def test_empty_string(self):
-        self.assertEqual(generate_soundex(""), "")
-
-    def test_single_character(self):
-        self.assertEqual(generate_soundex("A"), "A000")
-
-    
+    def test_get_soundex_code(self):
+        """Test the get_soundex_code function"""
```

Comment:
unwanted/repeated test cases ! Don't test get_soundex_code , As function is not a part of default interface